### PR TITLE
Configure frontend testing setup 

### PR DESCRIPTION
**Pull request by lin8231**
* Uses Jest and Enzyme for testing 
* Need to run `npm i` in frontend folder after pulling to install new dependencies 
* Currently no comprehensive testing, just setup 

More tests to come in later PR.

### CODE REVIEW

File: project/frontend/webpack.config.js [hunk 1]
```diff
@@ -33,8 +33,16 @@ module.exports = {
...
               {
-                plugins: ["@babel/plugin-proposal-class-properties"],
+                plugins: [
+                  [
+                    "@babel/plugin-proposal-class-properties",
+                    {
+                      loose: true,
+                    },
+                  ],
+                ],
```
Comment (roonil-wazlib) [inline project/frontend/webpack.config.js:45]: Is this actually how you're supposed to format this stuff? 😬

File: project/frontend/src/index.js [hunk 2]
```diff
@@ -1,22 +1,21 @@
-import "./index.css"
+import "./index.css";
 
-import React from "react"
-import ReactDOM from "react-dom"
-import {BrowserRouter} from "react-router-dom";
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter } from "react-router-dom";
```
Comment (texm) [inline project/frontend/src/index.js:5]: Import now unused

**Review by texm — APPROVED** [review]

File: project/frontend/webpack.config.js (hunk 1 above)
Comment (lin8231) [reply project/frontend/webpack.config.js:44]: Looks weird but it is yeah, or at least the formatter says so

File: project/frontend/src/index.js (hunk 2 above)
Comment (lin8231) [reply project/frontend/src/index.js]: Removed. Thanks!

**Review by roonil-wazlib — APPROVED** [review]

**Comment by roonil-wazlib** [comment]
LGTM

**Pull request merged**